tool/lint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Not sure this script belongs here, as it doesn't have anything pub-specific in it.

You could probably make a personal script that runs `dartfmt` and `dartanalyzer` in the current directory, and then run that on any dart project if it is needed for your workflow.

I personally have format-on-save in my editor to help me format.

Reply:
Yeah, it is more of a personal script and it may not fit in everone´s workflow.